### ISSUE TITLE
Why does percentage + percentage give the incorrect answer

### ISSUE THREAD

**Issue by hamodey** [josdejong/mathjs#2851]
Hi,

I am fairly new to using `mathjs` and I have been using it for my excel-like datasheets on my online application. 

However, recently I have seen an issue with evaluating decimal numbers, even the simplest expression does not output the correct answer, or what I believe should be correct. 

**Describe the bug**
```javascript
const expression = '10.00% + 20.00%'
const evaluation = math.evaluate(expression);
```

I am getting a weird output as follows: 
![image](https://user-images.githubusercontent.com/26579110/204779704-775b5411-e7bd-4e3f-b053-b188ec54d2a0.png)

Could I get some clarification pls?

Thanks :)

**Comment by josdejong** [comment]
The expression parser evaluates this expression as follows:

```js
math.evaluate('50%') 
// = 50 * 0.01
// = 0.5

math.evaluate('50% + 20%') 
// = (50 * 0.01) + 20%
// = 0.5 + 20%
// = 0.5 * 1.2
// = 0.6
```

If you want to get the result of 30%, you can provide parenthesis, like:

```js
math.evaluate('(10 + 20)%') 
// 0.3
```

The percentage operator is listed here in the docs: https://mathjs.org/docs/expressions/syntax.html

Does that help?

**Comment by gwhitney** [comment]
I think the original post does raise a legitimate issue that `10% + 20%` is at least ambiguous between adding the percentages and increasing the left operand by 20%. In fact I would say that the most common "everyday" interpretation of this expression would be to add the percentages and get 30%. So it seems as though perhaps it ought to be flagged as ambiguous or perhaps even be interpreted as the percentage addition. It's a real oddity that there isn't any parenthesization that disambiguates between these two interpretations-- note you had to remove one percent operator to get the desired 30% interpretation, with the current parser. I don't have a specific proposal offhand, but perhaps this should be moved to design discussions for further thought?

**Comment by josdejong** [comment]
Yes you're right, it would be much more intuitive to evaluate `10% + 20%` as `30%`. 

OK I'll move this into a discussion/idea. We'll have to think though how we can support this and in what cases exactly.
